output/components.eslint.txt

```diff
@@ -8,6 +8,9 @@ The react/require-extension rule is deprecated. Please use the import/extensions
 /home/travis/build/Talend/ui/packages/components/src/DateTimePickers/Date/Manager/Manager.component.js
   91:12  error  'onChange' PropType is defined but prop is never used  react/no-unused-prop-types
 
+/home/travis/build/Talend/ui/packages/components/src/DateTimePickers/DateRange/Manager/Manager.component.js
+  95:12  error  'onChange' PropType is defined but prop is never used  react/no-unused-prop-types
```

Comment:
Can you fix this lint error please ?

Reply:
https://github.com/Talend/ui/pull/2539#discussion_r345043717
seems eslint can't detect the usage.